src/components/app.tsx

```diff
@@ -329,11 +330,13 @@ export class App extends React.PureComponent<IProps, IState> {
             page={pagesVisible[currentPage - 1]}
             teacherEditionMode={teacherEditionMode}
             pluginsLoaded={pluginsLoaded}
-            glossaryPlugin={glossaryEmbeddable !== null}
+            plugins={embeddablePlugins !== null}
           />
         }
-        { !idle && glossaryEmbeddable && (activity.layout === ActivityLayouts.SinglePage || !isCompletionPage) &&
-          <GlossaryPlugin embeddable={glossaryEmbeddable} pageNumber={currentPage} pluginsLoaded={pluginsLoaded} />
+        { !idle && embeddablePlugins && (activity.layout === ActivityLayouts.SinglePage || !isCompletionPage) &&
```

Comment:
I see you made `getEmbeddablePlugins` return undefined instead of an empty array when there are no plugins found.  No biggee but if you used an empty array the `embeddablePlugins` reference in the conditional would no longer necessary as the map inside will return an empty array itself.